pom.xml

```diff
@@ -741,6 +742,12 @@
       <artifactId>simple-java-mail</artifactId>
       <version>5.0.0</version>
```

Comment:
Is there a reason we used a 16 month old version of `simple-java-mail`? It looks like `5.1.7` was released in May-19.

https://search.maven.org/search?q=g:org.simplejavamail%20AND%20a:simple-java-mail&core=gav

Reply:
(there could be a good reason; just curious)

Reply:
No good reason! Just copy-pasting.